cfme/tests/containers/test_relationships_tables.py

```diff
@@ -33,17 +31,26 @@
                           'Containers',
                           'Node'])
 def test_pods_rel(provider, rel):
+    """   This module verifies the integrity of the Relationships table
+          We also verify that clicking on the Relationships table field
+          takes the user to the correct page, and the number of rows
+          that appears on that page is equal to the number in the
+          Relationships table
+    """
     sel.force_navigate('containers_pods')
     list_tbl_pod = CheckboxTable(table_locator="//div[@id='list_grid']//table")
     ui_pods = [r.name.text for r in list_tbl_pod.rows()]
-    mgmt_objs = provider.mgmt.list_container_group()  # run only if table is not empty
+    ui_pods_revised = [
+        ch for ch in ui_pods if 'nginx' not in ch and not ch.startswith('metrics')]
```

Comment:
Can you please make a comment about why are the nginx and metrics pods so special that they need special handling here?